next-cloudinary/plugins/copy-assets.ts

```diff
@@ -0,0 +1,106 @@
+import { Plugin } from 'esbuild'
+import path from 'path';
+import { readdir, copyFile, readFile, lstat } from 'fs/promises';
+import { mkdirp } from 'mkdirp';
+
+let hasWrittenAssets = false;
+
+const assets = [
+  'cloudinary-video-player/dist/cld-video-player.min.css',
+  'cloudinary-video-player/dist/fonts'
+];
+
+export const plugin: Plugin = {
+  name: 'copy-assets',
+  setup: async () => {
+    const rootPath = path.join(__dirname, '../');
+    const distPath = path.join(rootPath, 'dist');
+
+    if ( hasWrittenAssets ) return;
+
+    await mkdirp(distPath);
+
+    for ( const asset of assets ) {
+      const assetPath = await resolveAssetPath(asset);
```

Comment:
i dont love this solution but couldn't figure out a way to resolve the node_modules reliably with being in a yarn workspace